05_syntax_lab/07.py

```diff
@@ -1,7 +1,17 @@
-""" Write a program that selects a random number
-and asks the user to guess it.
-After each guess print a hint "too large" or "too small" to the user.
-Bonus: To make things interesting, the program should cheat once in a white
-"""
-
-
+""" create random number between 0-100, and let the user guess it with our help"""
+from random import randint
+x = randint(1,100)
+y = 0
+i = 0
+while y != x:
+    print "please insert number:"
+    y = raw_input()
+    i += 1
+    if int(y) == x:
+        print "you did it! you guess that the number is %s in %s tries" %(x,i)
+        break

```

Comment:
There's a duplication here - either use `while True` and `break`, or drop the break as the while itself will not continue (because the above `while y != x`)